examples/federated-database-query-limit/README.md

```diff
@@ -5,24 +5,22 @@ This project aims to provide an example of using [MongoDB Atlas Federated Databa
 
 ## Dependencies
 
-* Terraform MongoDB Atlas Provider v1.10.0
+* Terraform MongoDB Atlas Provider v1.4.6
```

Comment:
This actually should be 1.10.0.   The fix on line 12 is indeed correct.

Reply:
updated